rpc_jobs/lem_multi_node_aio.yml

```diff
@@ -99,16 +102,22 @@
               tempest = load 'pipeline_steps/tempest.groovy'
               holland = load 'pipeline_steps/holland.groovy'
               maas = load 'pipeline_steps/maas.groovy'
+              kibana = load 'pipeline_steps/kibana.groovy'
           }}
           dir("rpc-gating/playbooks") {{
             writeFile file: "inventory/hosts", text: "[job_nodes]\n${{INSTANCE_NAME}} ansible_host=${{INSTANCE_IP}}\n"
             common.install_ansible()
           }}
           ssh_slave.connect()
           node(INSTANCE_NAME) {{
-            sh """
-               lvcreate -n aio -L250G lxc
-            """
+            common.conditionalStage(
```

Comment:
Did you consider adding this in a pipeline file? While this probably won't be used anywhere else it's likely nicer to read.